### DISCUSSION TITLE
Mullvad owned f-droid private repository

### DISCUSSION THREAD

**Discussion by serrq** [mullvad/mullvadvpn-app#8945] [Feature requests & Ideas]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

Unfortunately the f-droid main repo isn't updated with the last beta releases.

### Alternative solutions

Mullvad private repository

### Type of feature

- [ ] Better privacy/anonymity
- [ ] Better at circumventing censorship
- [X] Easier to use
- [ ] Other

### Operating System

- [X] Android
- [ ] iOS
- [ ] Windows
- [ ] macOS
- [ ] Linux

**Comment by albin-mullvad** [comment]
Hey! Thanks for the feature request! We've discussed this before, but will bring it up again internally for discussion.

Regarding this specific beta, it unfortunately got stuck on the fdroid build system not liking our way of storing the 3d map data. This means that `android/2024.1-beta1` will not reach fdroid, but instead next beta or stable will do so.

**Comment by Lolagatorade** [comment]
Hopefully the maintainer keeps the package up to date in FDroid but don't stress out about it. 

I use arch Linux and there's no native package. But someone on the AUR keeps a package up to date. Works fine even though it's not officially soupported on Arch.
